lightningd/bitcoind.c

```diff
@@ -814,12 +814,73 @@ static void fatal_bitcoind_failure(struct bitcoind *bitcoind, const char *error_
 	exit(1);
 }
 
+static char* get_blockchainname_for_cli(struct bitcoind *bitcoind)
+{
+	if (!strcmp(bitcoind->chainparams->network_name, "bitcoin")
```

Comment:
I believe we have a `streq` macro in `ccan/str/str.h` which matches this better.

Reply:
Yes, I fixed it.